view/frontend/web/js/view/payment/omise-offsite-method-renderer.js

```diff
@@ -0,0 +1,69 @@
+define(
+    [
+        'Omise_Payment/js/view/payment/omise-offline-method-renderer',
+        'Magento_Checkout/js/model/error-processor',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/model/url-builder',
+        'mage/storage',
+        'jquery'
+    ],
+    function (
+        Base,
+        errorProcessor,
```

Comment:
Do we need these 2 libs here? (`errorProcessor`, `fullScreenLoader`). Seems there is no place using it in this file.

Reply:
Well spotted - I thought I got all these 👍

Reply:
Well spotted - I thought I got all these 👍